### DISCUSSION TITLE
CarPlay carlink 4.0 cpc200-cp2a

### DISCUSSION THREAD

**Discussion by Teteris78** [tesla-android/issue-tracker#282] [General]
Hi, has anyone tried the 2023 carlink version instead of the 2022 one that is formally supported (cpc200-ccpa)? 
would it work? 
thx !

**Comment by mikegapinski** [comment]
cp2a -> carplay 2 air. Converts wired carplay into wireless. Not applicable to Tesla Android. CCPA is the newest supported dongle

**Reply by Teteris78** [reply]
Thanks, appreciate it. Will share more comments once I install on Tues !

**Comment by FMISSO** [comment]
Does anybody know if I can convert the Raspberry PICO W into a similar Carlink Dongle?
